elements/pfe-layouts/src/pfe-base.scss

```diff
@@ -24,17 +24,10 @@ h4,
 h5,
 h6 {
   padding: 0;
```

Comment:
Letting the browser defaults and/or custom styles create space between typography elements.

Reply:
When you pull this file into the demos, it breaks several components including card but it offers a lot of benefits too so I thought removing this small bit here would add more value to our demos than it would detract.